analyzers/src/SonarAnalyzer.VisualBasic/Rules/Hotspots/DeliveringDebugFeaturesInProduction.cs

```diff
@@ -51,7 +55,7 @@ public sealed class DeliveringDebugFeaturesInProduction : DeliveringDebugFeature
             node switch
             {
                 MultiLineIfBlockSyntax multiline => multiline.IfStatement.Condition,
-                SingleLineIfStatementSyntax singleline => singleline.Condition,
+                SingleLineIfStatementSyntax singleLine => singleLine.Condition,
```

Comment:
_Nitpick: _ This one was renamed to `singleLine` but the one above is `multiLine` - looks inconsistent now

Reply:
I will fix it in a following PR. I would not restart the pipeline just for that